src/lib/utils/options/readers/tsconfig.ts

```diff
@@ -26,6 +26,30 @@ export class TSConfigReader extends OptionsComponent
     private static OPTIONS_KEY:string = 'tsconfig';
 
 
+    private readConfigFile(event:DiscoverEvent, fileName:string) {
+        if (!FS.existsSync(fileName)) {
```

Comment:
@blakeembrey Not sure if [this check](https://github.com/Microsoft/TypeScript/blob/cf72ceb0933e0297ed9546cad3ae9def9916a114/src/compiler/commandLineParser.ts#L920) is needed here, what do you think?